openedx/core/djangoapps/discussions/models.py

```diff
@@ -20,12 +22,21 @@
 
 from openedx.core.djangoapps.config_model_utils.models import StackedConfigurationModel
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 
 log = logging.getLogger(__name__)
 
 DEFAULT_PROVIDER_TYPE = 'legacy'
 
 
+class ProviderMessages(NamedTuple):
```

Comment:
The same behavior can be achieved using 1 liner code. 

`ProviderMessages = namedtuple('ProviderMessages', ['messages'], defaults=([]))`

Reply:
Resolving this because [class was removed](https://github.com/edx/edx-platform/pull/28092#discussion_r676108553)